src/stan/mcmc/hmc/hamiltonians/ps_point.hpp

```diff
@@ -16,55 +16,33 @@ using Eigen::Dynamic;
  * Point in a generic phase space
  */
 class ps_point {
-  friend class ps_point_test;
-
  public:
-  explicit ps_point(int n) : q(n), p(n), V(0), g(n) {}
-
-  ps_point(const ps_point& z)
-      : q(z.q.size()), p(z.p.size()), V(z.V), g(z.g.size()) {
-    fast_vector_copy_<double>(q, z.q);
-    fast_vector_copy_<double>(p, z.p);
-    fast_vector_copy_<double>(g, z.g);
-  }
-
-  ps_point& operator=(const ps_point& z) {
-    if (this == &z)
-      return *this;
-
-    fast_vector_copy_<double>(q, z.q);
-
-    V = z.V;
-
-    fast_vector_copy_<double>(p, z.p);
-    fast_vector_copy_<double>(g, z.g);
-
-    return *this;
-  }
+  explicit ps_point(int n) : q(n), p(n), g(n) {}
 
   Eigen::VectorXd q;
   Eigen::VectorXd p;
-
-  double V;
   Eigen::VectorXd g;
+  double V{0};
 
-  virtual void get_param_names(std::vector<std::string>& model_names,
-                               std::vector<std::string>& names) {
-    for (int i = 0; i < q.size(); ++i)
-      names.push_back(model_names.at(i));
-    for (int i = 0; i < q.size(); ++i)
-      names.push_back(std::string("p_") + model_names.at(i));
+  virtual inline void get_param_names(std::vector<std::string>& model_names,
+                                      std::vector<std::string>& names) {
+    names.reserve(q.size() + p.size() + g.size());
     for (int i = 0; i < q.size(); ++i)
-      names.push_back(std::string("g_") + model_names.at(i));
+      names.emplace_back(model_names[i]);
```

Comment:
Here I accidentally removed the `std::string("g_");`. I made a revert pr so no worries!